src/confluent/azext_confluent/manual/commands.py

```diff
@@ -35,3 +35,5 @@ def load_command_table(self, _):
     with self.command_group('confluent terms', confluent_marketplace_agreement,
                             client_factory=cf_marketplace_agreement, deprecate_info=g.deprecate(redirect='az term', hide=True)) as g:
         g.custom_command('list', 'confluent_terms_list')
+    with self.command_group('confluent'):
```

Comment:
I mean you should remove this line.

Reply:
I have done this and pushed it. After that in the build checks

Reply:
@jsntcy : Can you help by retriggering the checks and see if the same error occurs in cicd pipeline ?